src/content/post/2025-07-29-finding-duplicates.mdx

```diff
@@ -0,0 +1,132 @@
+| juQ9Ve6wgOAk_Pnzmg-urzkk | EOSR2838                                              | d480a1345bd73d0c0e7e5c77a1e5ede29957996e | d480a1345bd73d0c0e7e5c77a1e5ede29957996e |
+| kw3Nqba5VexTMFjJwhKhuQAB | _R5_2647                                              | d482c009f15443283f4511eb74e88007d438277f | d482c009f15443283f4511eb74e88007d438277f |
+| MatEv_11w7hLcQRCBBGnvQZC | _R5_9965.jpg                                          | d4816ad416a14de431a5bcea6ea5c79231a4d94a | d4816ad416a14de431a5bcea6ea5c79231a4d94a |
+| SQBZAaAcyTXJOqRcrkrat0WV | _R5_6735                                              | d4858be893bf6049b6da785d64db9df9950f593e | d4858be893bf6049b6da785d64db9df9950f593e |
+| tqxyLA1aElH0nmiJnVe8rc0U | _R5_6024                                              | d48f65965a3083de9de961ea62ddfb635fa5b602 | d48f65965a3083de9de961ea62ddfb635fa5b602 |
+| vkDMj6_fgoTl3YEyalPtJLPD | DSC00289                                              | d482963a156577bf6612a08ed8a03f03c306e82c | d482963a156577bf6612a08ed8a03f03c306e82c |
+| VkIXrWJqPDK8B0bq36i95crq | _R5_7324.jpg                                          | d484a3e3510e6a86c5d8de84a23297d2eb136e19 | d484a3e3510e6a86c5d8de84a23297d2eb136e19 |
+| ZxEk1jsN5WWwUBE11BKy2rHY | _R5_2586                                              | d487d9e823856cdc48830de43241fc383ffe93cb | d487d9e823856cdc48830de43241fc383ffe93cb |
++--------------------------+-------------------------------------------------------+------------------------------------------+------------------------------------------+
+16 rows in set (0.001 sec)
+```
+
+We now have all the pictures where the `checksum` or `original_checksum` starts with `d48`, 16 matches.
+Of those only one is interesting for us, the one that matches exactly the sha1sum we computed before:
+`d48a6235db35f89b0ab27c9f460824d3ad7140be`. That one is `_r5_1287` with id `9aujfSLHxurI3mYQhizwvH3A`.
+
+Having the id of the image is not enough to locate it in the UI. We need to do a second query to figure out to which album
+it belongs to. Notice once again the `LIKE` and `%`.
+
+```sql
+SELECT * FROM photo_album WHERE photo_id LIKE '9aujfSLH%';
++--------------------------+--------------------------+
+| album_id                 | photo_id                 |
++--------------------------+--------------------------+
+| pmyOm8GBAUEI0auNGANJd-Uv | 9aujfSLHxurI3mYQhizwvH3A |
++--------------------------+--------------------------+
+1 row in set (0.035 sec)
+```
+
+We now know that our duplicate is located in album with id `pmyOm8GBAUEI0auNGANJd-Uv`.
+
+#### Step 5: Accessing the image.
+
+This is the final step and the easiest. Simply go to your website (`https://your-lychee-photo-gallery.test/gallery`) and login as admin to get all access rights.
+Then open `https://your-lychee-photo-gallery.test/gallery/<album_id>/<photo_id>`, this will directly put you in from of the duplicate in its album.
```

Comment:
```suggestion
Then open `https://your-lychee-photo-gallery.test/gallery/<album_id>/<photo_id>`, this will directly put you in front of the duplicate in its album.
```